### PR TITLE
Add missing image and fix grammar errors

### PR DESCRIPTION
**Pull request by BiancaPereira**
### Description

- Added missing image from the article (used the current github source from the original image).
- Fixed spelling error.

### Motivation

- Better reading of the article.

### Additional details

- Original article: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/How_CSS_works

### CODE REVIEW

File: files/pt-br/learn/css/first_steps/how_css_works/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Quando um navegador redenriza um documento, ele combina o documento com suas inf
 
 O diagrama a seguir também apresenta uma visão simples do processo.
 
-![](https://mdn.mozillademos.org/files/11781/rendering.svg)
+![Rendering process overview](https://github.com/mdn/content/blob/main/files/en-us/learn/css/first_steps/how_css_works/rendering.svg)
```
Comment (yin1999) [inline files/pt-br/learn/css/first_steps/how_css_works/index.md:54]: Hi @BiancaPereira, we can just use `rendering.svg` to refer the image file here, mdn/yari will fallback to find the image in en-US.

```suggestion
![Rendering process overview](rendering.svg)
```

Comment (BiancaPereira) [reply files/pt-br/learn/css/first_steps/how_css_works/index.md]: Thanks @yin1999!

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Comment by josielrocha** [comment]
Obrigado pela ajuda @BiancaPereira!

**Pull request merged**